### PR TITLE
Design system set up

### PR DESCRIPTION
**Pull request by tamzi**
# Scope

This is the Design system setUp  with empty libraries.

- [X] I have followed the coding conventions
- [X] I have added/updated necessary tests (OoS-Out of Scope)
- [X] I have tested the changes added on a physical device

## Closes/Fixes Issues
N/A

## Other testing QA Notes
YOu can run the project and switch between two apps: droidconApp and the chaiDemo App
Please add a screenshot (if necessary) 
N/A

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -13,6 +13,7 @@ plugins {
 
 buildscript {
 
+    val compose_version by extra("1.1.0-beta01")
```
Comment (kibettheophilus) [inline build.gradle.kts:16]: Remove this because it is not being used.

**Review by kibettheophilus — CHANGES_REQUESTED** [review]

**Review by wangerekaharun — APPROVED** [review]

File: settings.gradle.kts [hunk 2]
```diff
@@ -24,4 +24,6 @@ mapOf(
     project(":$projectName").projectDir = File(projectPath)
 }
 
-rootProject.name = "DroidconKE2022"
\ No newline at end of file
+rootProject.name = "DroidconKE2022"
+include(":chai")
```
Comment (wangerekaharun) [inline settings.gradle.kts:28]: You can add this on the `mapOf` block

File: build.gradle.kts (hunk 1 above)
Comment (tamzi) [reply build.gradle.kts]: done @kibettheophilus

File: settings.gradle.kts (hunk 2 above)
Comment (tamzi) [reply settings.gradle.kts]: Solved @wangerekaharun

**Review by kibettheophilus — APPROVED** [review]

**Pull request merged**